packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -196,6 +199,67 @@ router.get(
   })
 );
 
+// Zod schema to validate the request body based on the response of GET /duplicates
+const patchDuplicatesSchema = z.record(
+  // cx
+  z.record(
+    // patient
+    z.record(z.object({}).nullish())
+  )
+);
```

Comment:
The idea here is to facilitate the operation:
1. call `GET /duplicates`
2. copy the response payload
3. remove the persons that will be un-enrolled, leaving one person per patient
4. use that as the payload to `PATCH /duplicates/

Example response of `GET /duplicates/`:

```json
{
    "965f9957-34ac-442b-9a17-fa09c76c16da": {
        "018998ae-2677-78ab-8bb2-03369975d59f": {
            "06b33b36-4ef4-4d0a-9f13-3a1a48835329": {
                "amountOfLinks": 1,
                "enrolled": true,
                "enroller": "Org10",
                "enrollmentDate": "2023-06-14T20:25:41.164Z"
            },
            "48f5756a-000b-47eb-a29e-cc412ca3b2e6": {
                "amountOfLinks": 1,
                "enrolled": true,
                "enroller": "Org10",
                "enrollmentDate": "2023-08-11T23:08:01.371Z",
                "isMetriport": true
            },
            "3fa02bf5-ad0c-4f31-8fc7-653106a25a9a": {
                "amountOfLinks": "unknown",
                "enrolled": false,
                "enroller": "Metriport",
                "enrollmentDate": "2023-08-11T23:19:34.601Z"
            }
        }
    }
}
```

Example payload to `PATCH /duplicates/`:

```json
{
    "965f9957-34ac-442b-9a17-fa09c76c16da": {
        "018998ae-2677-78ab-8bb2-03369975d59f": {
            "06b33b36-4ef4-4d0a-9f13-3a1a48835329": {
            }
        }
    }
}
```